backbone/src/main/java/org/researchstack/backbone/step/QuestionStep.java

```diff
@@ -26,6 +31,11 @@
 
     private String placeholder;
 
+    /* Default constructor needed for serilization/deserialization of object */
```

Comment:
(Comment is about this class, but not code you changed)

Why does getStepLayoutClass() return a `Class`, and getStepBodyClass() return a `Class<?>`